src/aot/aot_main.cpp

```diff
@@ -6,9 +6,11 @@
 
```

Comment:
Is there a reason we can't push the changes required here into `run_bpftrace` or some core runtime layer? May require some gentle refactoring.